### ISSUE TITLE
IE11 not sending requests with Range

### ISSUE THREAD

**Issue by mikemorton** [videojs/video.js#3105]
I'm finding that when a video loads in IE11, it makes a single request that streams back the whole file.  It does not send the Range header.  Therefore, when the user clicks into the middle of the timeline to jump ahead, they end up seeing a spinner and have to wait for the video to buffer completely to that point.

Forgive me if this is a common request, but I did some searching around and couldn't find anyone else that encountered this problem. :smile:

**Comment by gkatsev** [comment]
Hi, can you please provide more information on your setup? What version of videojs? Any videojs plugin? What type of media are you trying to play? If you can provide a reduced test case, that would be even better.

**Comment by mmcc** [comment]
Also, if you're seeing this with just an mp4, you should test it out with a vanilla HTML5 video element (one without Video.js initialized on it).

**Comment by mikemorton** [comment]
Thanks for the quick responses.

I'm using 5.5.1, no plugins and mp4 files.

Tested with the vanilla video element and it has the same issue.  I guess that means the issue isn't in your code...

**Comment by gkatsev** [comment]
Sounds like an IE bug. You can file it against them on connect.microsoft.com but I don't think they'll going to be fixing IE11 bugs. However, this bug may be present in MS Edge, which they may fix.
Closing this as a browser bug.

**Comment by mikemorton** [comment]
Thanks, further research indicates it's not a bug but just one of their nice little quirks: http://stackoverflow.com/questions/25654422/http-pseudo-streaming-in-ie11

**Comment by gkatsev** [comment]
Interesting. Good to know. Thanks.

**Comment by wsteele** [comment]
I had the same problem until after some test I figured out that the initial request from IE/Edge does not include a range header.  If you correspondingly respond with an initial Content-Range of the file size equal to your Content-Length header IE will assume you just want to send the whole file in one go.  If you don't get an range header you are better off choosing some content length that is less than the size of the file.  I chose 1Mb, but it doesn't really matter I think.
